app/portainer/views/custom-templates/custom-templates-view/customTemplatesViewController.js

```diff
@@ -205,6 +205,25 @@ class CustomTemplatesViewController {
 
     template.Selected = true;
 
+    let file;
+    try {
+      if (template.GitConfig !== null) {
+        try {
+          file = await this.CustomTemplateService.fetchFileFromGitRepository(template.Id);
```

Comment:
after reading this a few times (on EE repo and here). why don't we just change `customTemplateFile` to fetch the file from git if there's a git config?

Reply:
fixed